radis/db/classes.py

```diff
@@ -216,6 +216,10 @@
 HITRAN_MOLECULES = list(trans.values())
 """ str: list of [HITRAN-2020]_ molecules. """
 
+def is_molecule(species):
+    if '_I' in species :
+        return True
```

Comment:
Looks wrong to me. Do you mean "is_neutral"?
What if there is '_II'. It will also return True

Reply:
Add unitary tests for this function

Reply:
You are right,  the name should rather be is_atom

Reply:
you can add a new test file in `radis\test\db\test_classes` . For instance

```python
assert is_atom("Fe_I")
assert is_atom("H_II")
assert not is_atom("CO2")

```


Btw, look at the function `radis.test.db.test_molecules.generate_molparam_for_non_HITRAN_species()` . I can't remember who implemented it and why but it has a `_parse_molecule` regex

Reply:
Thank you I had a look at the regex. 
I think if we want to adjust it at some point so that it can also identify atomic species, it should be _parse_molecule = re.compile("(\d+)?([A-Z][a-z]?)_?([A-Z][a-z\d+]?)?") with the optional _? for the ionization state
However, is_atom can be enough to distinguish atoms or ions from molecules so it depends on what the goal is precisely @erwanp

Reply:
If is_atom is enough, keep it !
It would be nice (= must not) to use physical conventions, for instance `Fe` rather than `Fe_I` or `Fe+` rather than `Fe_II`. To simplify the code a simple converter could be added only at the higher level, for instance converting the input of `calc_spectrum`, and making sure every function within the code then uses chemical conventions. 

For instance a renaming dictionary: 
```
_species_rename = {'Ar': 'Ar_I',
                   'Ar I': 'Ar_I',
                   'AR': 'Ar_I',  # Cantera uses capital only
                   'Ar+': 'Ar_II',
                   'Ar II': 'Ar_II',
                   'C': 'C_I',
                   'C I': 'C_I',
                   'C+': 'C_II',
                   'C II': 'C_II',
                   'O': 'O_I',
                   'O I': 'O_I',
                   'O+': 'O_II',
                   'O II': 'O_II',
                   'N': 'N_I',
                   'N I': 'N_I',
                   'N+': 'N_II',
                   'N II': 'N_II',
                   }
```

Reply:
Just to be sure I understood what you meant: the spectroscopic notation (`Fe_I`) should be the default notation in RADIS. These notations should be what the code primarily uses to distinguish the atoms and ions. Let's not add confusion by adding new names. 

However, it would be nice for the user interface only, that when asking `calc_spectrum(Fe)`, RADIS would understand that we meant `Fe_I`.

Reply:
yes exactly

Reply:
I have added a to_conventional_name method to radis/db/classes below is_atom
If the input of calc_spectrum is an atomic species in a format such as Na or Na+ it will be modified to Na_I , Na_II etc
Molecules will remain the same
I have also added some test in radis/test/db/test_classes.py

Reply:
The checks fail when SpectrumFactory is used with molecule instead of species because of the is_atom call 
radis/lbl/factory.py:508: in __init__
    if not is_atom(species):
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
species = None
    def is_atom(species):
>       if '_I' in species :
E       TypeError: argument of type 'NoneType' is not iterable
For instance 
You can see the build report it happens in severals files @erwanp

Reply:
Sorry I'm on my phone and I cannot fully check the problem but here, "species" should be a list of species name. Not a None